gulpfile.js

```diff
@@ -51,12 +64,22 @@ gulp.task('js', function() {
         .pipe(gulp.dest('data/js'));
 });
 
-gulp.task('watch', ['scss', 'js'], function() {
-    gulp.watch('src/scss/**/*.scss', ['scss']);
+// lint JS files
+gulp.task('js:lint', function() {
+    return gulp.src(['src/js/**/*.js', '!src/js/**/*.min.js'])
+        .pipe(eslint())
+        .pipe(eslint.format())
+        .pipe(eslint.failAfterError());
+});
+
+
+gulp.task('watch', ['scss', 'scss:lint', 'js', 'js:lint'], function() {
+    gulp.watch('src/scss/**/*.scss', ['scss', 'scss:lint']);
     gulp.watch('src/js/**/*', ['js']);
+    gulp.watch(['src/js/**/*.js', '!src/js/**/*.min.js'] ['js:lint']);
```

Comment:
Ah! This is the problem - there needs to be a comma between these arrays. After adding, I get JS linter notices while gulp is running.